src/Tribe/PUE/Checker.php

```diff
@@ -2209,7 +2283,47 @@ public static function monitor_active_plugins( Tribe__PUE__Checker $checker ): v
 					$transient_data['plugins'][ $plugin_slug ] = $checker->is_key_valid();
 				}
 			}
+		}
+
+		/**
+		 * Initializes a license check during PUE Checker initialization.
+		 *
+		 * @param Tribe__PUE__Checker $checker An instance of the PUE Checker.
+		 */
+		public function initialize_license_check( Tribe__PUE__Checker $checker ): void {
```

Comment:
Where do we use this?

Reply:
I'm glad you pointed that out. I must of accidentally removed it when I was merging Imp into the branch. Technically that method is the entire reason for this PR :facepalm: